### ISSUE TITLE
Topology: Error on click of node in backstage instance

### ISSUE THREAD

**Issue by invincibleJai** [patternfly/react-topology#44]
**Describe the problem**
Getting console error on node click in topology view in the backstage instance, this happens likely as there are different versions of d3 and its dependencies i.e  d3-selection library is present with 3.0.0 and pf react-topology has 1.4.2

**How do you reproduce the problem?**

1. checkout main https://github.com/janus-idp/backstage-plugins (tried on commit `a70889f3304125368e948f15a87f67ae84ab296e`)
2. cd backstage-plugins 
3. rm -rf node_modules (if already have project cloned then remove node_modules)
4. yarn install
5. yarn workspace @janus-idp/backstage-plugin-topology run start
6. select topology from nav and click on any node

error is shown below

<img width="1791" alt="Pasted Graphic 1" src="https://user-images.githubusercontent.com/5129024/233556722-14c7e79b-88f6-425d-b10e-b700253d1672.png">

**Expected behavior**
No error should be shown

**Is this issue blocking you?**
List the workaround if there is one.

**Screenshots**
If applicable, add screenshots to help explain the issue.

**What is your environment?**
- OS: [e.g. iOS] mac OS
- Browser [e.g. chrome, safari] chrome
```
node --version
v16.17.1
yarn -v
1.22.0

"@patternfly/react-topology": "4.93.4",
```

**What is your product and what release date are you targeting?** 
Backstage / Janus-IDP, this will impact all the plugins consuming pf react-topology

**Any other information?**

**Comment by jeff-phillips-18** [comment]
This is an application issue with pulling in multiple versions of `d3-selection`. Currently we use D3 version 5 which pulls in d3-selection version 1. You are pulling in d3-selection version 3. These versions are quite different and it will take some time to upgrade PF react-topology to version 3.

**Comment by jeff-phillips-18** [comment]
Fixed by #59
